src/crewai/cli/plus_api.py

```diff
@@ -25,7 +25,9 @@ def __init__(self, api_key: str) -> None:
 
     def _make_request(self, method: str, endpoint: str, **kwargs) -> requests.Response:
         url = urljoin(self.base_url, endpoint)
-        return requests.request(method, url, headers=self.headers, **kwargs)
+        session = requests.Session()
+        session.trust_env = False
```

Comment:
`trust_env` tells `requests` to bypass the system's authentication or proxy settings, relying only on the authentication defined in the request itself.

The reason for this change is that when a `.netrc` file is present, `requests` automatically replaces the `Authorization` header with the credentials from `.netrc`, overwriting Auth0 credentials.

While `trust_env = False` fixes the issue, alongside I'm proposing moving our custom PyPI index endpoints from `app.crewai.com` to `tools.crewai.com`. This ensures `.netrc` is only applied to that specific subdomain system-wide.

Reply:
Your suggestion makes total sense.